### PR TITLE
Fixed: visualization settings dialog broken if press 'V' key

### PR DESCRIPTION
**Pull request by afedchin**
second commit it's a little improvement to not doing restart addon if it has no settings.

@MartijnKaijser ping someone who responsible for this part

### CODE REVIEW

File: system/keymaps/joystick.Harmony.xml [hunk 1]
```diff
@@ -209,7 +209,7 @@
       <!-- Prev		-->      <button id="32">LockPreset</button>
       <!-- Info  	-->      <button id="31">Info</button>
       <!-- F8		-->      <button id="96">ActivateWindow(VisualisationPresetList)</button>
-      <!-- F9		-->      <button id="73">ActivateWindow(VisualisationSettings)</button>
+      <!-- F9		-->      <button id="73">Addon.Default.OpenSettings(xbmc.player.musicviz)</button>
```
Comment (MartijnKaijser) [inline system/keymaps/joystick.Harmony.xml]: from what i can recall is you can drop the "xbmc."  part. Same in the other cases

**Comment by MartijnKaijser** [comment]
not sure who to ping on this

**Comment by Paxxi** [comment]
possibly @mkortstiege and @topfs2 ?

File: system/keymaps/joystick.Harmony.xml (hunk 1 above)
Comment (afedchin) [inline system/keymaps/joystick.Harmony.xml]: without xbmc. it doesn't work

**Comment by mkortstiege** [comment]
Not my department, but changes look fine to me.

**Comment by MartijnKaijser** [comment]
we'll find out
jenkins build and merge

**Pull request merged**